community/examples/AMD/hpc-cluster-amd-slurmv5.yaml

```diff
@@ -65,20 +65,46 @@ deployment_groups:
       - type: shell
         source: modules/startup-script/examples/install_ansible.sh
         destination: install_ansible.sh
+      - $(swfs.install_nfs_client_runner)
+      - $(swfs.mount_runner)
       - $(spack.install_spack_deps_runner)
       - $(spack.install_spack_runner)
       # the following installation of AOCC may be automated in the future
       # with a clear direction to the user to read the EULA at
       # https://developer.amd.com/aocc-compiler-eula/
+      - type: shell
+        content: "shutdown -h +1"
+        destination: shutdown.sh
+
+  - id: slurm_startup
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - type: data
+        destination: /etc/profile.d/spack.sh
+        content: |
+          #!/bin/sh
+          if [ -f /sw/spack/share/spack/setup-env.sh ]; then
+                  . /sw/spack/share/spack/setup-env.sh
+          fi
       - type: data
         destination: /var/tmp/install_aocc.sh
         content: |
           #!/bin/bash
+          source /sw/spack/share/spack/setup-env.sh
           spack install aocc@3.2.0 +license-agreed
           spack load aocc@3.2.0
           spack compiler find --scope site
           spack -d install -v openmpi@4.1.3 %aocc@3.2.0 +legacylaunchers +pmi schedulers=slurm
 
+  # must restart vm to re-initiate subsequent installs
+  - id: spack-builder
```

Comment:
nit: The IDs of other modules are using underscores rather than dashes.

Reply:
I will change that. Personal opinion: no dashes in IDs since they will be picked up by TFLint as a no-no.

Reply:
Should be done.